docs/core/versions/index.md

```diff
@@ -1,101 +1,226 @@
 ---
-title: .NET Core Versioning
-description: Learn about the versioning scheme for .NET Core.
-keywords: .NET, .NET Core
-author: richlander
+title: .NET Core versioning
+description: Understand how 
```

Comment:
@BillWagner Ha, good catch! I was filing a bug for Gauntlet regarding descriptions and forgot to add this back. LOL